test/JDBC/upgrade/16_1/schedule

```diff
@@ -496,3 +496,4 @@ permission_restrictions_from_pg
 BABEL-4529-before-15_6-or-14_11
 BABEL-730-before-15_6-or-16_1
 babel-4475
+babel-4517
```

Comment:
Please add this to `latest/schedule` file as well.

Reply:
OK, missed that one, thanks for the finding